cmd/activity-monitor/main.go

```diff
@@ -151,14 +146,12 @@ func main() {
 			syslogU, err := url.Parse(c.GlobalString("jsonlog"))
 			if err != nil {
 				log.Fatalf("Could not parse Syslog URL: %s", err)
-				return
 			}
 
 			logger.SetEndpoint(syslogU.Scheme, syslogU.Host)
 			err = logger.Connect()
 			if err != nil {
 				log.Fatalf("Could not open remote syslog: %s", err)
-				return
```

Comment:
log.Fatalf() just prints a log message at the "fatal" level.  It doesn't change the control flow at all.  Thus we need to keep the returns so that we stop executing.